pom.xml

```diff
@@ -2734,6 +2734,8 @@
         <module>cdap-security-spi</module>
         <module>cdap-security</module>
         <module>cdap-storage-spi</module>
+        <module>cdap-system-app-api</module>
+        <module>cdap-system-app-unit-test</module>
```

Comment:
why do unit tests need to be in their own module?

Reply:
Do you mean instead of just in cdap-unit-test? We didn't want system tables to be accessible in normal unit tests

If you mean instead of putting them in the api module, it's to keep api free from all the dependencies that would get pulled in.

Reply:
Your first statement makes sense.
Let's go with what you have, then.